code/game/objects/items/defibrillator.dm

```diff
@@ -72,8 +72,8 @@
 
 	defib_cooldown = world.time + 2 SECONDS
 	ready = !ready
-	user.visible_message("<span class='notice'>[user] turns [src] [ready? "on and takes the paddles out" : "off and puts the paddles back in"].</span>",
-	"<span class='notice'>You turn [src] [ready? "on and take the paddles out" : "off and put the paddles back in"].</span>")
+	user.visible_message("<span class='notice'>[user] turns [src] [ready? "on and opens the cover" : "off and puts the paddles back in"].</span>",
```

Comment:
```suggestion
	user.visible_message("<span class='notice'>[user] turns [src] [ready? "on and opens the cover" : "off and closes the cover"].</span>",
```

Reply:
Please add a space after `ready` and before the `?`